### ISSUE TITLE
View and functionality of XLIFF tab (FE)

### ISSUE THREAD

**Issue by MariusJulius** [keeleinstituut/tv-tolkevarav#349]

**Comment by IngaSokman** [comment]
@PlaksoBirgit I have a question about saving the XLIFF to selected sub-tasks, just for clarification - can the user edit the selection as many times as he/she wants? Also, for example if I split the task to two vendors, I add all the XLIFF files under one vendor, do I undestand correctly that the other vendor will not receive then XLIFF files but will receive source files?

**Comment by PlaksoBirgit** [comment]
Yes, incase split tasks it's possible that some vendor is without any XLIFF's (but they will receive the source files) - example: there might be an order where part of the files are translated with CAT tool, but some are not (bad quality) and then it's possible to leave a note to that particular vendor e.g to take the pdf files and translate only these and other part is done by another vendor(s) via cat tool

**Comment by IngaSokman** [comment]
@PlaksoBirgit Got it. But please confirm the editing of the selection as well. How many times can the user change the selection, is it until it has been sent to vendors? If the selection has been made, after saving of the data the new changes are displayed (what has been selected and what not) right? If the user starts to change them again then firstly I'll show the last changes the user has made?

**Comment by PlaksoBirgit** [comment]
"How many times can the user change the selection, is it until it has been sent to vendors?" - no, user can change also after it has been sent, see 5.2.1 in PM wiki -> 
![image](https://github.com/keeleinstituut/tv-tolkevarav/assets/107627103/410a3e32-807e-413b-a5af-cc4d6eee18e8)

"If the selection has been made, after saving of the data the new changes are displayed (what has been selected and what not) right?" - yes
"If the user starts to change them again then firstly I'll show the last changes the user has made?" yes, if the user has saved the previous changes, then we show the last saved state

**Comment by IngaSokman** [comment]
@PlaksoBirgit Just to clarify - in the wiki there is said "By default all the files (one or many) are checked for the "Teostaja 1"" - do I understand correctly that if the user doesn't change the default values, these values (XLIFF files) will be sent to the Teostaja 1 after pressing "Saada alamtellimus teostajale"?

**Comment by PlaksoBirgit** [comment]
yes, that is correct (reason is that we do not want to make some extra complicated logic, the PM is responsible for managing the XLIFFs)

**Comment by plakitkelly** [comment]
OK
